client/src/App.js

```diff
@@ -1,17 +1,22 @@
-import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import React from 'react';
+import { MuiThemeProvider } from '@material-ui/core';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+import { theme } from './themes/theme';
+import Login from './pages/Login';
+import SignUp from './pages/SignUp';
 
-import "./App.css";
+import './App.css';
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Switch>
+          <Route path="/signup" component={SignUp} />
+          <Route path="/login" component={Login} />
+          <Route path="/" exact component={SignUp} />
```

Comment:
We might discuss with the team removing the `exact` from the root path to have a catch-all signup page.

Reply:
I think when we will have our user authorized, we will render dashboard (homepage) on '/' route, otherwise redirect to SignUp